content/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/configuring-commit-squashing-for-pull-requests.md

```diff
@@ -20,9 +20,14 @@ shortTitle: Configure commit squashing
 
 {% data reusables.repositories.navigate-to-repo %}
 {% data reusables.repositories.sidebar-settings %}
-1. Under "Pull Requests", select **Allow squash merging**. This allows contributors to merge a pull request by squashing all commits into a single commit. The default commit message presented to contributors when merging is the commit title and message if the pull request contains only 1 commit, or the pull request title and list of commits if the pull request contains 2 or more commits. {% ifversion ghes = 3.6 %} To always use the title of the pull request regardless of the number of commits in the pull request select **Default to PR title for squash merge commits**.{% endif %}
+1. On the "General" tab (which is selected by default), scroll down to the section marked "Pull Requests".
+![Screenshot 2023-08-31 at 2 16 53 PM](https://github.com/github/docs/assets/59002/fc027e4c-62f5-4abd-8530-050fa24060a3)
+
+
+
+   Select **Allow squash merging**. This allows contributors to merge a pull request by squashing all commits into a single commit. The default commit message presented to contributors when merging is the commit title and message if the pull request contains only 1 commit, or the pull request title and list of commits if the pull request contains 2 or more commits. {% ifversion ghes = 3.6 %} To always use the title of the pull request regardless of the number of commits in the pull request select **Default to PR title for squash merge commits**.{% endif %}
```

Comment:
```suggestion
1. On the "General" settings page (which is selected by default), scroll down to the section marked "Pull Requests". 
1. Under "Pull Requests", select **Allow squash merging**. This allows contributors to merge a pull request by squashing all commits into a single commit. The default commit message presented to contributors when merging is the commit title and message if the pull request contains only 1 commit, or the pull request title and list of commits if the pull request contains 2 or more commits. {% ifversion ghes = 3.6 %} To always use the title of the pull request regardless of the number of commits in the pull request select **Default to PR title for squash merge commits**.{% endif %}
```

Reply:
I don't think the screenshot is strictly necessary in this case (we tend to use them sparingly and I think your proposed clarification to the content is sufficient!). ✨

I'm also suggesting a slight edit to the structure - in general, keeping one action per step.